kubernetes-examples/src/main/java/io/fabric8/openshift/examples/BuildConfigExamples.java

```diff
@@ -37,10 +37,8 @@
 
   public static void main(String[] args) throws InterruptedException {
     Config config = new ConfigBuilder().build();
-    KubernetesClient kubernetesClient = new DefaultKubernetesClient(config);
-    OpenShiftClient client = kubernetesClient.adapt(OpenShiftClient.class);
-
-    try {
+    try(KubernetesClient kubernetesClient = new DefaultKubernetesClient(config)){
+      OpenShiftClient client = kubernetesClient.adapt(OpenShiftClient.class);
```

Comment:
@manusa or is the issue caused because we have moved this `client` variable inside the `try` block?

Reply:
As rohan menitoned, you don't need to explicitly close the client now that it is in a try-with-resources block

Reply:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html